netebpfext/net_ebpf_ext.h

```diff
@@ -236,17 +236,15 @@ typedef struct _net_ebpf_extension_flow_context_parameters
 
 typedef enum _net_ebpf_extension_hook_id
 {
-    EBPF_HOOK_OUTBOUND_L2 = 0,
-    EBPF_HOOK_INBOUND_L2,
     EBPF_HOOK_ALE_RESOURCE_ALLOC_V4,
     EBPF_HOOK_ALE_RESOURCE_ALLOC_V6,
     EBPF_HOOK_ALE_RESOURCE_RELEASE_V4,
-    EBPF_HOOK_ALE_RESOURCE_RELEASE_V6, // 5
```

Comment:
I think adding comments to reflect the enum values in increments of 5 is helpful.